lib/install/config/webpacker.yml

```diff
@@ -4,6 +4,7 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
+  compile: false
```

Comment:
Since compilation only happens when packs are missing (vs. stale), the name `compile` doesn't feel quite right. Perhaps something more descriptive like `compile_missing_packs`?